crates/egui/src/containers/window.rs

```diff
@@ -33,20 +33,21 @@ use super::{area, resize, Area, Frame, Resize, ScrollArea};
 /// Note that this is NOT a native OS window.
 /// To create a new native OS window, use [`crate::Context::show_viewport_deferred`].
 #[must_use = "You should call .show()"]
-pub struct Window<'open> {
+pub struct Window<'open, 'collapsed> {
```

Comment:
Could maybe be same lifetimes for simplicitly, though since they can technically be different they should perhaps be modelled like this.

I think any usecase I would have they would have the same lifetimes though